_pages/mobile-interview-project.md

```diff
@@ -182,6 +182,10 @@ Definitely! Please utilize as many Google searches and Stack Overflow searches a
 
 That being said, please do not bulk copy and paste large sections of code from 3rd party sources, as this would defeat the purpose of the evaluation. If you do copy in code, please note as such in your README.
 
+Q) **Can I use GitHub Copilot or other AI code generation tools?**
+
+No. We don't use GitHub Copilot or other AI code generation tools at Block, and we want this project to reflect the same tools and techniques that you would use on the job.
```

Comment:
> or other AI code generation tools

I don't know if this is true, or if it will remain true in the near future.

Reply:
wanted to leave it open-ended for now. Copilot is the preeminent tool, but there are others that are currently working on [similar tools](https://github.com/CodedotAl/gpt-code-clippy).

Reply:
We can always update this when it changes.